src/ufw.yml

```diff
@@ -6,36 +6,46 @@
   roles:
     - ufw
   vars:
-    # Do not actually enforce firewall rules.  The assessors will turn
-    # UFW on if they want it.
-    state: disabled
+    # Enable ufw and enforce firewall rules by default.
+    state: enabled
 
 - hosts: all
   name: Basic configuration for UFW
   become: yes
   become_method: sudo
   tasks:
-    - name: Allow incoming traffic through default TCP ports
+    - name: Allow incoming traffic through specified TCP ports
       community.general.ufw:
         proto: tcp
         rule: allow
         to_port: "{{ item }}"
       loop:
         - "22"       # ssh for Guacamole server (internal to assessment env)
         - "25"       # Mail (external)
-        - "53"       # C2 traffic (external)
-        - "80"       # Domain verification (external)
-        - "443"      # Payload hosting (external)
         - "587"      # Mail (external)
         - "993"      # IMAP (internal to assessment env)
+        - 5000:5999  # Additional listeners, if necessary (external)
         - "5901"     # vnc for Guacamole (internal to assessment env)
+        - "50050"    # CobaltStrike (internal to assessment env)
+    - name: Deny incoming traffic to specified TCP ports
+      # These rules may be manually modified by assessors as needed, but we
+      # want to default to a minimal set of allowed ports.
+      community.general.ufw:
+        proto: tcp
+        rule: deny
+        to_port: "{{ item }}"
+      loop:
+        - "53"       # C2 traffic (external)
+        - "80"       # Domain verification (external)
+        - "443"      # Payload hosting (external)
         - "8080"     # C2 listening and testing (external)
         - 8000:8999  # Additional listeners, if necessary (external)
```

Comment:
My understanding was that they are using 5000-5999 now _instead of_ 8000-8999, in which case this line can be removed.

Reply:
Hello all, for the TeamServer you are correct, however since NOM reversed the change that was requested a while back our TeamServers currently still have port range 8000-8999 exposed to the world. So in an abundance of caution I've included that be part of the deny list until it's formally removed from the build process.

Reply:
@dav3r - If the 8000-8999 port range stays then please add a comment explaining [what @m1j09830 stated above](https://github.com/cisagov/teamserver-packer/pull/52#discussion_r796924230).

Reply:
@jsf9k after speaking with @dav3r about it I went ahead and removed that range from the initial ticket. I'm going to coordinate with NOM to implement our original request of removing ingress over that port range now that you all have implemented the internal range for us to use. I've updated the original ticket to reflect the change.

Reply:
I spoke with @m1j09830 and he agreed that we will no longer need to include 8000-8999 here, so I will remove it.  He has already updated [his comment in cisagov/cool-system-internal#80](https://github.com/cisagov/cool-system-internal/issues/80#issuecomment-1025951812).

Reply:
I removed that port range in https://github.com/cisagov/teamserver-packer/pull/52/commits/4d0cf1401da251c048cf040cee7e1aba73dd5d0f.